docs/user-guide/src/irso/introduction.md

```diff
@@ -79,7 +79,7 @@ values for each branch and release of the operator:
 
 | Operator version | Ironic version(s) | Default version |
 | ---------------- | ----------------- | --------------- |
-| latest (main)    | 27.0              | latest          |
+| latest (main)    | 28.0, 27.0        | latest          |
```

Comment:
We do have two IRSO releases, should we be documenting those?

Reply:
I've updated it to include the released versions